doc/manual/manual.tex

```diff
@@ -5597,11 +5597,11 @@ \subsubsection{Simple convection in a quarter of a 2d annulus}
 \begin{equation*}
   \textrm{Ra}
   =
-  \frac{g\; \beta \; \Delta T \; \rho \; L^3}{\alpha\eta}
-  \approx
-  \frac{10 \text{m}\,\text{s}^{-2} \; 4\cdot 10^{-5}\text{K}^{-1} \; 3300
-  \text{K} \; 3300 \text{kg}\,\text{m}^{-2} \; (2.86 \cdot 10^6
-  \text{m})^3}{10^{22}
+  \frac{g\, \alpha  \Delta T  \rho  L^3}{\kappa\eta}
+=
+  \frac{10\, \text{m}\,\text{s}^{-2} \times 4\cdot 10^{-5}\text{K}^{-1} \times 3300\,
+  \text{K} \times 3300\, \text{kg}\,\text{m}^{-3} \times (2.86 \cdot 10^6
+  \text{m})^3}{10^{-6}\, \text{m}^2\,\text{s}^{-1}\times 10^{22}\,
```

Comment:
Looking at this again, it turns out that the only non-obvious part is the addition of the factor `10^6 m^2/s`. This is because the original formula was simply missing it -- it corresponds to the thermal diffusivity kappa.